src/main/java/home/intexsoft/bank_application/attributeDescriptor/AttributeDescriptor.java

```diff
@@ -0,0 +1,35 @@
+package home.intexsoft.bank_application.attributeDescriptor;
+
+public class AttributeDescriptor {
+
+    private DescriptorParameter kind;
+    private String value;
+
+    public AttributeDescriptor(DescriptorParameter kind, String value) {
+        this.kind = kind;
+        this.value = value;
+    }
+
+    public DescriptorParameter getKind() {
+        return kind;
+    }
+
+    public String getValue() {
+        return value;
+    }
+
+    public enum DescriptorParameter {
+        MAX_VALUE,
+        MIN_VALUE,
+        TYPE;
+
+        public void setDescriptorParameterName(String descriptorParameterName) {
```

Comment:
@DmitryZagorski, something strange is going on here, we are normally using constructor to set enum constants attrs.